src/str.js

```diff
@@ -1056,7 +1066,11 @@ Sk.builtin.str.prototype.nb$remainder = function (rhs) {
         }
         base = 10;
         if (conversionType === "d" || conversionType === "i") {
-            return handleWidth(formatNumber(value, 10));
+            let tmpData = formatNumber(value, base);
```

Comment:
wouldn't it make more sense to call `.toFixed(0)` on this? instead of doing the `slice` below?

Reply:
`tmpData ` is a `string array`, if want to use `toFixed(0)` need to make the sting to be a number.
Like use `parseInt, parseFloat, Number etc`.. Normally, it's work fine for small number. But for big  number which like '"123456789127123546890"` , that's can't work. So i use slice to operate the `string`

Reply:
Alright I hadn't thought of that

Reply:
Yes, at the beginning I used `parseInt etc`, but can't pass all tests.
So I decided use this way.
:smile:

Reply:
got it.

Reply:
looks good.